### DISCUSSION TITLE
Support for Chrome Dev Protocols

### DISCUSSION THREAD

**Discussion by taruntechno** [seleniumbase/SeleniumBase#1208] [Q&A] [answered]
Does seleniumbase supports chrome dev protocols for eg . View console logs and get network traffic and performance.

**Comment by mdmintz** [comment, marked as answer]
Hi @taruntechno , yes, as seen from examples such as https://github.com/seleniumbase/SeleniumBase/blob/master/examples/user_agent_test.py , you can run CDP commands using ``self.driver.execute_cdp_cmd()``. Example:

```python
self.driver.execute_cdp_cmd(
    "Network.setUserAgentOverride", {
        "userAgent": "Mozilla/5.0 "
                     "(Nintendo Switch; WifiWebAuthApplet) "
                     "AppleWebKit/606.4 (KHTML, like Gecko) "
                     "NF/6.0.1.15.4 NintendoBrowser/5.1.0.20393"
    }
)
```

**Reply by taruntechno** [reply]
Thanks a lot
